easybuild/easyconfigs/b/BOLT-LMM/BOLT-LMM-2.4.1-iimkl-2024a.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'MakeCp'
+
+name = 'BOLT-LMM'
+version = '2.4.1'
+
+homepage = 'https://storage.googleapis.com/broad-alkesgroup-public/BOLT-LMM/BOLT-LMM_manual.html'
+description = """The BOLT-LMM algorithm computes statistics for testing association between phenotype
+ and genotypes using a linear mixed model (LMM) [1]. By default, BOLT-LMM assumes a Bayesian
+ mixture-of-normals prior for the random effect attributed to SNPs other than the one being tested.
+ This model generalizes the standard "infinitesimal" mixed model used by previous mixed model
+ association methods (e.g., EMMAX, FaST-LMM, GEMMA, GRAMMAR-Gamma, GCTA-LOCO), providing an
+ opportunity for increased power to detect associations while controlling false
+ positives. Additionally, BOLT-LMM applies algorithmic advances to compute mixed model association
+ statistics much faster than eigendecomposition-based methods, both when using the Bayesian mixture
+ model and when specialized to standard mixed model association."""
+
+toolchain = {'name': 'iimkl', 'version': '2024a'}
+
+# only latest sources are available under `/downloads`, older releases are in the `old` subdir
+source_urls = [
+    'https://alkesgroup.broadinstitute.org/BOLT-LMM/downloads/',
+    'https://alkesgroup.broadinstitute.org/BOLT-LMM/downloads/old',
+]
+sources = ['%(name)s_v%(version)s.tar.gz']
+checksums = ['65f13a51277a0834e03911a9e98c9325be95d1d896a3e7b8d2aa759776a78aa6']
+
+builddependencies = [
+    ('zstd', '1.5.6'),
```

Comment:
`zstd` is already pulled by the dependency on `Boost`

Reply:
hmm... why is it a full dependency in `Boost`? I imagine it only needs to be a build dep there too

Reply:
ah, good point.